modules/machineset-capacity-reservation.adoc

```diff
@@ -0,0 +1,161 @@
+// Module included in the following assemblies:
+//
+// * machine_management/creating_machinesets/creating-machineset-azure.adoc
+// * machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-azure.adoc
+
+ifeval::["{context}" == "creating-machineset-azure"]
+:azure:
+endif::[]
+ifeval::["{context}" == "cpmso-config-options-azure"]
+:azure:
+endif::[]
+ifeval::["{context}" == "creating-machineset-aws"]
+:aws:
+endif::[]
+ifeval::["{context}" == "cpmso-config-options-aws"]
+:aws:
+endif::[]
+
+:_mod-docs-content-type: PROCEDURE
+[id="machineset-capacity-reservation_{context}"]
+ifdef::azure[= Configuring Capacity Reservation by using machine sets]
+ifdef::aws[= Configuring Capacity Reservations by using machine sets]
+
+{product-title} version {product-version} and later supports using
+ifdef::azure[on-demand Capacity Reservation with Capacity Reservation groups on {azure-full} clusters.]
+ifdef::aws[Capacity Reservations on {aws-full} clusters, including On-Demand Capacity Reservations and Capacity Blocks for ML.]
+
+You can configure a machine set to deploy machines on any available resources that match the parameters of a capacity request that you define.
+These parameters specify the VM size, region, and number of instances that you want to reserve.
```

Comment:
not sure if `VM size` need some update, because on Azure, it's [vmSize](https://drive.google.com/file/d/1G7ZuHG_FbcccksezU0EtwXYHR7YUjJza/view?usp=sharing), but on AWS, it's [instanceType](https://drive.google.com/file/d/16mSa-GeJub7dkmzK-71TR_UhcuahmQr6/view?usp=sharing).

Reply:
thanks!